<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductAddDialog</class>
 <widget class="QDialog" name="ProductAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>452</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>539</width>
    <height>452</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>539</width>
    <height>452</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EasyManager-ProductAdd</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background-image:url(':/Theme/BackImage/DialogBack_1.jpg');
}
QLabel
{
font:bold 15px;
}
QSpinBox
{
font:bold 15px
}</string>
  </property>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>商品名称</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商品进货单价</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="InPriceEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商品数量</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="CountSpinBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>计划出售单价</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutPriceEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商品详细信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InfoEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ImageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileUrlEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>图像文件选择：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FileChooseButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmButton">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>362</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>确认添加该商品</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
